rust_src/src/callint.rs

```diff
@@ -74,4 +80,17 @@ use remacs_macros::lisp_fn;
 #[lisp_fn(min = "0", unevalled = "true")]
 pub fn interactive(_args: LispObject) {}
 
```

Comment:
maybe `from compiled code` instead of `from C`? Or `from outside of lisp`?

Reply:
fixed